eu.betterbird.Betterbird.desktop

```diff
@@ -0,0 +1,88 @@
+[Desktop Entry]
```

Comment:
this file is upstream, isn't it? Because a desktop app without a .desktop file is a bug.

Reply:
Mozilla family is [legendary](https://bugzilla.mozilla.org/show_bug.cgi?id=296568) for not shipping a desktop file so this fork continues the legend :smile:

Reply:
so, "better" it is not.

Reply:
The .desktop file will be provided by upstream and removed from this repo.
https://github.com/Betterbird/thunderbird-patches/issues/68